app/static/js/submitform.js

```diff
@@ -0,0 +1,14 @@
+function make_request() { 
+    let queryField = document.getElementById('queryField').value;
+    let limitField = document.getElementById('limitField').value;
+    const http = new XMLHttpRequest();
+    const queryUrl = encodeURIComponent(`${queryField}`)
```

Comment:
Can ``${queryField}`` just be `queryField`? Same question with `limitField`

Reply:
yeah